README.md

```diff
@@ -62,6 +62,32 @@ python -m expertise.tfidf_scores config.json
 
 The output will generate a `.csv` file with the name pattern `<config_name>-scores.csv`.
 
+## Detect Duplicates
+
+There are two modes for duplicate detection: intra-conference and inter-conference. Intra-conference detection detects duplicate submissions within the same conference and inter-conference detection detects duplicate submissions with other conferences. For intra-conference detection only one `submissions` directory or one `submissions.jsonl` file is needed. For inter-conference a `submissions` directory and a `other_submissions` directory or a `submissions.jsonl` file and a `other_submissions.jsonl` file are needed. More details about these files/directories can be found at the end of this section.
```

Comment:
@carlosmondra how about this:

Duplicate detection can be used to find duplicates both within a venue or between 2 different venues. 
- For detecting duplicates within a venue only one `submissions` directory or one `submissions.jsonl` file is needed.
- For detecting duplicated between 2 different venues, either one `submissions` and one `other submissions` directories are needed **or** one `submissions.jsonl` and one `other_submissions.jsonl` are needed.

More details about these files/directories can be found at the end of this section.

Reply:
That definitely sounds better haha. I have updated the README.